examples/advanced_circuits_algorithms/Quantum_Phase_Estimation/Quantum_Phase_Estimation.ipynb

```diff
@@ -224,8 +224,8 @@
    "outputs": [],
```

Comment:
<p>Did we run this notebook again? I think the cost to run the example should be 0.00 USD since we are running on Local Simulator. Everything else looks good!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>